packages/lib-classifier/src/components/Classifier/components/SubjectViewer/components/InteractionLayer/components/TranscribedLines/TranscribedLines.spec.js

```diff
@@ -279,101 +218,94 @@ describe('Component > TranscribedLines', function () {
     })
 
     describe('on click', function () {
-      let annotation, consensusLines, line, result, user
-
-      beforeEach(function () {
+      let consensusLines, line, user
+
+      beforeEach(async function () {
+        const store = mockStore({ client, workflow: workflowSnapshot })
+        render(<TranscribedLines />, { wrapper: withStore(store) })
+        const subject = store.subjects.active
+        await when(() => subject.caesarReductions?.reductions.length > 0)
+        task = store.workflowSteps.findTasksByType('transcription')[0]
+        task.setActiveTool(0)
         user = userEvent.setup({ delay: null })
         task.reset()
-        consensusLines = transcriptionReductions.consensusLines(0)
-        annotation = {
-          taskKey: 'T1',
-          taskType: 'transcription',
-          update: sinon.stub(),
-          value: []
-        }
-        const store = mockStore({ workflow: workflowSnapshot })
-        result = render(<TranscribedLines
-          annotation={annotation}
-          frame={0}
-          lines={consensusLines}
-          marks={task.marks}
-          task={task}
-        />, { wrapper: withStore(store) })
+        consensusLines = store.subjects.active.caesarReductions.consensusLines(0)
         line = document.querySelector('g.transcribed.line')
       })
 
       it('should create a new mark', async function () {
         const transcribedLines = consensusLines.filter(line => !line.consensusReached)
         expect(task.marks.length).to.equal(0)
         expect(task.activeMark).to.be.undefined()
+        expect(line.getAttribute('aria-disabled')).to.equal('false')
+        expect(line.tabIndex).to.equal(0)
         await user.click(line)
-        expect(task.activeMark.x1).to.equal(transcribedLines[0].points[0].x)
-        expect(task.activeMark.y1).to.equal(transcribedLines[0].points[0].y)
-        expect(task.activeMark.x2).to.equal(transcribedLines[0].points[1].x)
-        expect(task.activeMark.y2).to.equal(transcribedLines[0].points[1].y)
-        expect(task.activeMark.frame).to.equal(transcribedLines[0].frame)
-        expect(task.marks.length).to.equal(1)
-        task.setActiveMark(undefined)
+        line = document.querySelector('g.transcribed.line')
+        expect(line.getAttribute('aria-disabled')).to.equal('true')
+        expect(line.tabIndex).to.equal(-1)
+        // expect(task.marks.length).to.equal(1)
+//         expect(task.activeMark).to.exist()
+//         expect(task.activeMark.x1).to.equal(transcribedLines[0].points[0].x)
+//         expect(task.activeMark.y1).to.equal(transcribedLines[0].points[0].y)
+//         expect(task.activeMark.x2).to.equal(transcribedLines[0].points[1].x)
+//         expect(task.activeMark.y2).to.equal(transcribedLines[0].points[1].y)
+//         expect(task.activeMark.frame).to.equal(transcribedLines[0].frame)
+//         task.setActiveMark(undefined)
```

Comment:
I can't see why these tests fail. If I inspect `task.marks` inside the `TranscribedLines` component, after the line is clicked, it contains a new mark, which is active. If I inspect `task.marks` here, in the tests, its still an empty array.

Reply:
These tests are clicking a disabled line, which does nothing.